flux/src/image-automation/ImageAutomationList.tsx

```diff
@@ -73,7 +84,14 @@ export default function ImageAutomationWrapper() {
 
   if (!isImageAutomationControllerAvailable) {
     return (
-      <SectionBox>
+      <SectionBox  style={{
+        padding: '1rem',
+        alignItems: 'center',
+        margin: '2rem auto',
+        height: '20vh',
+        width: '50%',
```

Comment:
This may mess the width when in narrow screens. Please either account for smaller screens, or use `maxWidth: '600px'` for example.